src/scripts/keyboardNavigation.js

```diff
@@ -239,12 +290,13 @@ function attachGamepadScript() {
 }
 
 // No need to check for gamepads manually at load time, the eventhandler will be fired for that
-if (navigator.getGamepads && appSettings.enableGamepad()) {
+if (navigator.getGamepads && appSettings.enableGamepad() && canEnableGamepad()) {
     window.addEventListener('gamepadconnected', attachGamepadScript);
 }
 
 export default {
     enable: enable,
     getKeyName: getKeyName,
-    isNavigationKey: isNavigationKey
+    isNavigationKey: isNavigationKey,
+    canEnableGamepad: canEnableGamepad
```

Comment:
You can use the shorthand syntax for these:

```suggestion
    isNavigationKey,
    canEnableGamepad
```